test/hack/bin/get-inhibition.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+GIT_WORKDIR="$(git rev-parse --show-toplevel)"
+REPO_VERSION="$(curl -s https://api.github.com/repos/giantswarm/prometheus-meta-operator/releases/latest | jq -r .name)"
+
+curl https://raw.githubusercontent.com/giantswarm/prometheus-meta-operator/"$REPO_VERSION"/files/templates/alertmanager/alertmanager.yaml > "$GIT_WORKDIR"/test/hack/checkLabels/alertmanager.yaml
```

Comment:
I don't think the progress bar is very useful here; I'd rather have it less verbose.
```suggestion
curl -s https://raw.githubusercontent.com/giantswarm/prometheus-meta-operator/"$REPO_VERSION"/files/templates/alertmanager/alertmanager.yaml > "$GIT_WORKDIR"/test/hack/checkLabels/alertmanager.yaml
```

Reply:
Added